resources/js/features/game-list/components/DataTableFacetedFilter/DataTableFacetedFilter.tsx

```diff
@@ -82,14 +82,14 @@ export function DataTableFacetedFilter<TData, TValue>({
           size="sm"
           className={cn(
             'border-dashed',
-            buildTrackingClassNames(`Click ${title} Filter`),
+            buildTrackingClassNames(`Click ${t_title} Filter`),
             className,
           )}
-          data-testid={`filter-${title}`}
+          data-testid={`filter-${t_title}`}
```

Comment:
This also seems like something that might be better off using a non-translated string. Perhaps even a string without spaces.

Reply:
Agreed, I've also changed this in latest to now use the column ID.